src/odemis/driver/npmc.py

```diff
@@ -294,6 +294,9 @@ def _findDevice(self, ports, baudrate=19200):
 
                 try:
                     ve = self.GetVersion()
+                    if not re.match(r"ESP301 Version \d\.\d\.\d (1[0-2]|[1-9])\/(3[01]|[12][0-9]|[1-9])\/(99|[0-9]?[0-9])", ve):
```

Comment:
Let's be a little bit more relaxed on the match. I'd say that if `ve.upper()` contains ESP301, it's fine.

Reply:
Haha... I got a bit carried away with the Regex I think.